src/CoreWCF.Primitives/src/CoreWCF/Security/SecurityHeaderTokenResolver.cs

```diff
@@ -0,0 +1,329 @@
+using System;
+using System.Globalization;
+using System.IO;
+using System.Security.Cryptography.X509Certificates;
+using CoreWCF.IdentityModel;
+using CoreWCF.IdentityModel.Selectors;
+using CoreWCF.IdentityModel.Tokens;
+using CoreWCF.Security.Tokens;
+
+namespace CoreWCF.Security
+{
+    internal sealed class SecurityHeaderTokenResolver : SecurityTokenResolver // , System.IdentityModel.IWrappedTokenKeyResolver
```

Comment:
Remove commented out interface implementation as it's not actually used anywhere